### DISCUSSION TITLE
Product List Default Order not working

### DISCUSSION THREAD

**Discussion by cmdigitaluk** [openstore-ecommerce/OpenStore#282] [Q&A]
I've seen a couple of posts on this subject but none of them seemed to provide an answer.  I want to sort my product list alphabetically when you navigate to the page.  I've edited ProductDisplayAjaxList.cshtml to

@AddPreProcessMetaData("orderby", "NB3.ProductName", Model.FullTemplateName, Model.ModuleId.ToString())
@AddPreProcessMetaData("orderby0", "NB3.ProductName", Model.FullTemplateName, Model.ModuleId.ToString())

I've commented out the other orderby lines but but I still can't get my product list to show in the correct order

**Comment by DNNMonster** [comment]
I think this is a first render issue where the orderby hidden input is empty and the ProductFunctions code path to allow the sorting is not getting executed.  I pushed a branch that adds the first render check and should fix this.

**Comment by leedavi** [comment]
It's also confusing sometimes, because the "Manual Order" is sometimes turned on for the list.  This means that the products list list in a manual sort order for a category.  So if you select a category, it might not be in alphabetical order.  This is an option in the store config settings, in the admin panel.

@AddPreProcessMetaData("orderby", "NB3.ProductName", Model.FullTemplateName, Model.ModuleId.ToString())

Is usually the only line needed if you don't want the user to select the order.
